examples/fundamentals.test.js

```diff
@@ -4355,6 +4355,34 @@ describe('Fundamentals', function() {
       // acquit:ignore:end
     });
   });
+    it('NaNs', function() {
+      isNaN(NaN); //true
+      isNaN(Number.NaN); //true
+      Number.isNaN(NaN); //true
+      let text = 'hello world!';
+      if (isNaN(text)) {
+        console.log(text);
+        text = '1';
+      }
+      if (isNaN(text)) {
```

Comment:
Also worth mentioning the difference between `isNaN()` and `Number.isNaN()`: https://masteringjs.io/tutorials/fundamentals/nan